51-android.rules

```diff
@@ -891,7 +904,7 @@ LABEL="cdc", ENV{adb_user}="yes", GOTO="android_usb_rule_match"
 LABEL="adbfast", ENV{adb_adb}="yes"
 LABEL="fast", ENV{adb_fast}="yes", GOTO="android_usb_rule_match"
 
-# ADB Debug and mass storage (note: android_ver<4.3)
+# ADB Debug and mass storage (note: generally android_ver<4.3; Moto Z3 Play when powered off)
```

Comment:
Oh, and I forgot to ask. Is the mass storage meant for this case too, or should it be a separate LABEL?